src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -1110,18 +1110,26 @@ private void HandleReadResponseContentException(Exception ex, CancellationToken
                     throw new IOException(SR.net_http_client_execution_error, new HttpRequestException(SR.net_http_client_execution_error, abortException));
                 case Http3ConnectionException _:
                     // A connection-level protocol error has occurred on our stream.
-                    _connection.Abort(ex);
+                    _connection?.Abort(ex);
```

Comment:
When can this happen? It seems weird that errors on a single HTTP stream would cause the whole connection to be killed.